specs/pvs/corla.pvs

```diff
@@ -438,8 +472,10 @@ BEGIN
 
 % @description A decision to be made, either the selection of one or
 % more candidates for an office or the choice of a course of action.
+% @todo kiniry Need to clarify choices vs options vs votes.
   ballot_contest: TYPE = contest
```

Comment:
What thing of type "contest" would not be a "ballot_contest"? (Or perhaps I've misunderstood the syntax and this is just saying that "contest" and "ballot_contest" are synonyms?